actors/builtin/miner/policy.go

```diff
@@ -32,19 +38,23 @@ func init() {
 	}
 }
 
-// The maximum number of sectors that a miner can have simultaneously active.
-// This also bounds the number of faults that can be declared, etc.
+// PARAM_SPEC
+// The maximum number of sectors that a miner can have (and implicitly the maximum number of faults, recoveries, etc.)
+// Note that when sectors are terminated they count towards SectorsMax until cleanup.
+// Motivation: This guarantees that actors operations per miner are bounded.
 // TODO raise this number, carefully
 // https://github.com/filecoin-project/specs-actors/issues/470
 const SectorsMax = 32 << 20 // PARAM_FINISH
 
-// The maximum number of partitions that may be required to be loaded in a single invocation.
+// PARAM_SPEC
+// The maximum number of partitions that can be loaded in a single invocation.
 // This limits the number of simultaneous fault, recovery, or sector-extension declarations.
-// With 48 deadlines (half-hour), 200 partitions per declaration permits loading a full EiB of 32GiB
+// With 48 deadlines (half-hour), 238 partitions per declaration permits loading a full EiB of 32GiB
 // sectors with 1 message per epoch within a single half-hour deadline. A miner can of course submit more messages.
-const AddressedPartitionsMax = 200
+const AddressedPartitionsMax = 238
 
-// The maximum number of sector infos that may be required to be loaded in a single invocation.
+// PARAM_SPEC
+// The maximum number of sector infos that can be loaded in a single invocation.
 const AddressedSectorsMax = 10_000
```

Comment:
**Clarification request**:
The calculation for `AddressedPartitionsMax` is correct (we have however updated it with the new blocktime).

However, `AddressedSectorsMax` seems incorrect. If a miner can only report a maximum of `10000` sectors per DeclareFault (for example) and send one message per epoch, it would take them ~30h to declare 1EiB faulted (calculation: 10000 sectors * 32GiB * (60hours*30*60/30).

Reply:
A miner can submit many messages per epoch, limited only by competition for block gas limit.

Reply:
Why is this set to 10000? The previous number had a motivation (see calculation in comments), I can’t find a valid motivation for this one.

Reply:
It's a good question. It's a made up number that seems large enough but hopefully not too large. A better answer would be "empirically determined", but we haven't done the necessary benchmarking.

Reply:
Ok, we can do some of that